main/src/test/scala/com/dwolla/fs2aws/s3/S3ClientSpec.scala

```diff
@@ -100,14 +96,70 @@ class S3ClientSpec(implicit ee: ExecutionEnv) extends Specification with Matcher
       client.downloadObject(bucket, key, ee.ec).compile.toList must returnValue(equalTo(expected.toList))
     }
 
-    "delete object from the given S3 location" >> {
+    "delete object from the given S3 location" in new setup {
       client.deleteObject(bucket, key).compile.drain must returnOk
 
       there was one(mockS3).deleteObject(bucket, key)
     }
+
+    "list the keys in the given bucket when the results are not truncated" in new setup {
+      val listing = new ObjectListing()
+      val summary1 = new S3ObjectSummary()
+      summary1.setKey("key1")
+      listing.getObjectSummaries.add(summary1)
+      when(mockS3.listObjects(any[ListObjectsRequest])).thenAnswer { invo: InvocationOnMock =>
```

Comment:
I forgot that SAM inlining is only available in Scala 2.12, but this library is still published for Scala 2.11 as well. We'll need to change this to explicitly create an anonymous subclass of `Answer` in order for it to compile on 2.11.